Modify rule S6258: Improve description (#510)
SonarSource/rspec: rules/S6258/message.adoc

=== Message

> Make sure that disabling logs for this component won't affect your security detection capabilities.

Reviewer: There is also a generic format that we use for hotspot messages. You can find it [here](https://docs.sonarqube.org/latest/extend/adding-coding-rules/) at the very end.

So here the message should look like this:
```suggestion
Make sure that disabling logging is safe here.
```
